### DISCUSSION TITLE
Some ideias for Believers Sword

### DISCUSSION THREAD

**Discussion by pepaulo** [JenuelDev/Believers-Sword#34] [Ideas]
The Believers Sword app has a amazing future in the Study Bible section. This is my tiny ideias. 

* Change the Sermons item to Study pad, a Very good way to study Bible and make Sermons. 
* Read of the ALL books in the MyBible modules. At the moment is read only Protestant Canon. 
* Read of Commentaries Modules. 
* Read the Bible text in Full View mode. Or without  de sidebar View. 
* Read the ALL Modules in the download directory. This can to possibility add other modules who is out of Believers Sword repository.

**Comment by JenuelDev** [comment]
I will going to consider the first 4, but the fifth, the SQLite table should be the same with the format we are using.. 😁

**Reply by pepaulo** [reply]
Merry Christmas. I believe I expressed myself badly in this fifth suggestion. What I wanted to say was the possibility of the application recognizing the .SQLite3 files placed in the directory other than those downloaded by the application. These .SQLite3 modules would also be displayed in the list of Bibles available to the user.

**Reply by JenuelDev** [reply]
ah I see,, ok ok, I get it now. I will also consider it,, going to add it in the to do list as well. :)
